rewards/aws/boost.py

```diff
@@ -65,6 +80,18 @@ def download_boosts(chain: str):
     return data
 
 
+def download_proposed_boosts(chain: str):
```

Comment:
Return typing here please

Reply:
This is gonna return  a dict  - in another PR we can create dataclasses for the boost and tree dicts